### PR TITLE
Readable credential names in the profile

### PR DESCRIPTION
**Pull request by huwshimi**
Fixes: #2711.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- connect guiproxy to a jimm with real models
- open your profile
- click on a model
- the credential name should be just the name

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7079/

File: jujugui/static/gui/src/app/store/env/controller-api.js [hunk 1]
```diff
@@ -565,6 +565,7 @@ YUI.add('juju-controller-api', function(Y) {
             controllerUUID: result['controller-uuid'],
             owner: tags.parse(tags.USER, result['owner-tag']),
             credential: credential,
+            credentialName: this._parseCredentialName(credential),
```
Comment (frankban) [inline jujugui/static/gui/src/app/store/env/controller-api.js]: Please update the docstring to reflect the presence of the new field.

File: jujugui/static/gui/src/app/store/env/controller-api.js [hunk 2]
```diff
@@ -679,6 +680,7 @@ YUI.add('juju-controller-api', function(Y) {
               controllerUUID: model.controllerUUID,
               owner: model.owner,
               credential: model.credential,
+              credentialName: this._parseCredentialName(model.credential),
```
Comment (frankban) [inline jujugui/static/gui/src/app/store/env/controller-api.js]: Ditto.

**Comment by frankban** [comment]
QA ok.

File: jujugui/static/gui/src/app/store/env/controller-api.js (hunk 1 above)
Comment (huwshimi) [inline jujugui/static/gui/src/app/store/env/controller-api.js]: What the what? I did actually add the docstrings. What happened to them?! I guess I'll add them again :).

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7083/

**Comment by huwshimi** [comment]
Thanks @frankban @hatched :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**